feat: Adding contract addresses (#164)
matter-labs/zksync-docs: content/00.build/95.resources/80.addresses.md

---
title: Addresses
description: Page with deployment addresses
---

# Mainnet

Bridgehub: https://etherscan.io/address/0x303a465B659cBB0ab36eE643eA362c509EEb5213

SharedBridge: 0xD7f9f54194C633F36CCD5F3da84ad4a1c38cB2cB

## STM (official)
STM: 0xc2eE6b6af7d616f6e27ce7F4A451Aedc2b0F5f5C

Chains:

| name | chain_id  |  description   | address  | Explorer url |
| --- | --- | --- | --- | --- |
| ZKsync Era | 324 | zksync era mainnet | 0x32400084c286cf3e17e7b677ea9583e60a000324 | https://etherscan.io/address/0x32400084c286cf3e17e7b677ea9583e60a000324

# Testnet

Bridgehub:  https://sepolia.etherscan.io/address/0x35A54c8C757806eB6820629bc82d90E056394C92
SharedBridge: https://sepolia.etherscan.io/address/0x3E8b2fe58675126ed30d0d12dea2A9bda72D18Ae

## STM (official)
STM: 0x4e39E90746A9ee410A8Ce173C7B96D3AfEd444a5

> Hyperchains:

Reviewer: ```suggestion
ZK Chains:
```

| name | chain_id  |  description   | address  | Explorer url |
| --- | --- | --- | --- | --- |
| ZKsync Sepolia testnet  | 300 | zksync era testnet | 0x9A6DE0f62Aa270A8bCB1e2610078650D539B1Ef9 | https://sepolia.etherscan.io/address/0x9A6DE0f62Aa270A8bCB1e2610078650D539B1Ef9 |
| ?? | 11124 | ?? | 0x8aD52ff836A30f063dF51A00C99518880B8b36ac | |
| ?? | 444 | ?? | 0x2F45Fc02ec23e6D335fdE7B5a5083F053C8C2aD3 | |
| ?? | 531050104 | ?? | 0xDd02368f378EeD3779B358D2020a77e4D022236D | |
| GRVT | 326 | Gravity Sepolia testnet | 0x0e4Bd2E8D953a0234Fb1122FFB848B49522308eC | |
| ?? | 327 | ?? | 0xB484A3F98bCbAAf50d66292463203c0247B4f3F6 | |

## STM (custom - Cronos)
STM: 0x762b5F15CAd9880ace81776f9046d6a52DD67a9b

> Hyperchains:

Reviewer: ```suggestion
ZK Chains:
```

| name | chain_id  |  description   | address  | Explorer url |
| --- | --- | --- | --- | --- |
| Cronos testnet  | 282 | Chronos testnet | 0xeBf961DC3904725ec916678D875B9625d5F7C29f | |

# How to get this data.

Start from Bridgehub.
To get the STM, call stateTransitionManager (and pass chain id)

> in stateTransitionManager, you can query the StateTransition (a.k.a Hyperchain) contract.

Reviewer: ```suggestion
in stateTransitionManager, you can query the StateTransition (a.k.a ZK Chain) contract.
```
